File: client/check_braces.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\yasha_ambulkar\\OneDrive\\Documents\\26-03-26\\MAIN\\client\\src\\pages\\EvaraFlowAnalytics.tsx', 'utf8');
const lines = content.split('\n');
let level = 0;
for (let i = 0; i < lines.length; i++) {
    const line = lines[i];
    for (let char of line) {
        if (char === '{') level++;
        if (char === '}') level--;
    }
}
console.log(`Final level: ${level}`);

File: client/check_braces.py
import sys

path = r'c:\Users\yasha_ambulkar\OneDrive\Documents\26-03-26\MAIN\client\src\pages\EvaraFlowAnalytics.tsx'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

level = 0
for i, char in enumerate(content):
    if char == '{':
        level += 1
    elif char == '}':
        level -= 1
    if level < 0:
        print(f"Negative level at char {i}")
        break

print(f"Final level: {level}")
